src/main.jsx

```diff
@@ -18,6 +19,10 @@ ReactDOM.createRoot(document.getElementById('root')).render(
         {/* Landing Pages */}
         <Route path="/" element={<Landing />} />
         <Route path="/courses" element={<Courses />} />
+        <Route path="/course/software-engineering" element={<Course/>}/>
```

Comment:
Not like this it should be like this

\<Route path="/course/:course" element={\<Course course=course />}/>

[https://reactrouter.com/en/main/hooks/use-params](https://reactrouter.com/en/main/hooks/use-params) check this out

Reply:
https://refine.dev/blog/react-router-useparams/

Reply:
fixed this